### ISSUE TITLE
Discord Channel

### ISSUE THREAD

**Issue by QuantaZ1** [meta-boy/Anime-Twist#8]
Since you are developing this App, I suggest you to have a Discord channel through which you can communicate with your users. Also I had a question, when are you releasing the App?

**Comment by meta-boy** [comment]
There already exists an official android app for Anime Twist, this is just a flutter clone of the app for cross-platform support. There also exists official [Anime Twist Discord Server](https://discord.gg/2k7awVP), you can find me there as well as more info on the app there.

About releasing, yes I will release a stable build in this coming week with more features which are mentioned in #1

**Comment by QuantaZ1** [comment]
Will you maintain this App or just using it to learn and then abandon it?

**Comment by meta-boy** [comment]
Yeah, I plan to maintain it, but won't publish it anywhere. Will only make releases for android as `.apk` is distributable. I have planned several things for the future, Will make this app not just for watching anime but for all anime related things :)

**Comment by QuantaZ1** [comment]
That would be really great. But, hey, since you also have it for iOS, why not have it available on TestFlight or just make the IPA available on GitHub too, so that users can sideload it. :)

**Comment by meta-boy** [comment]
Sure I'll do that once I get my Mac machine all set up.

**Comment by QuantaZ1** [comment]
Great
